spynl/main/__init__.py

```diff
@@ -1,17 +1,25 @@
 """The main package of Spynl."""
 
-
-import os.path
+from pkg_resources import iter_entry_points  # pylint: disable=E0611
 
 from pyramid.config import Configurator
-from pyramid.settings import asbool
 from pyramid.security import NO_PERMISSION_REQUIRED
 from pyramid.viewderivers import INGRESS
 
-from spynl.main import (plugins, serial, about,
-                        routing, events, endpoints, session)
-from spynl.main.utils import (renderer_factory, check_origin,
-                              handle_pre_flight_request)
+from spynl.main import (
+    serial,
+    about,
+    plugins,
+    routing,
+    events,
+    endpoints,
+    session,
+)
+from spynl.main.utils import (
+    renderer_factory,
+    check_origin,
+    handle_pre_flight_request,
+)
```

Comment:
I personally prefer it in a way that it is easy to add, remove, comment imports. It helps when debugging and developing. Hence my changes.

But I can revert it.